backend/README.md

```diff
@@ -103,3 +103,11 @@ Note however that this results in a) all history being saved in memory instead o
 
 We use [state migrations](./src/database/state-migrations/) to convert outdated states to new versions.
 Look at [`migrations.ts`](./src/database/state-migrations/migrations.ts) for more information.
+
+### Note on long term storage
+
+The current setup when using a database is that no exercises get deleted unless anyone deletes them from the UI (or, to be more precisely, using the HTTP request `DELETE /api/exercises/:exerciseId`).
+An average exercise with four segments and about 45 minutes of exercise time seems to take up about 10 MB storage in the database.
```

Comment:
```suggestion
An average exercise with four viewports being actively played and about 45 minutes of exercise time seems to take up about 10 MB storage in the database.
```

instead of viewport you could also use "section", which is the domain term rather than the technical term.